src/azul/service/hca_response_v5.py

```diff
@@ -29,7 +31,7 @@ class TermObj(JsonObject):
     Class defining a term object in the FacetObj
     """
     count = IntegerProperty()
-    term = StringProperty()
+    term = Union[StringProperty(), DictProperty()]
```

Comment:
Nah, this is confounding the `typing` and `jsonobject` packages. They are not meant to be combined this way.